### PR TITLE
[OSDOCS-8650]: Content review of Understanding the Cluster Network Operator topic

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8650

Link to docs preview:
https://69684--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/cluster-network-operator

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@danwinship  Are you able to review from a SME perspective, or recommend someone who can?
@zhaozhanqi Are you able to QE review these doc updates?

Thanks!

File: modules/nw-operator-cr.adoc [hunk 1]
```diff
@@ -419,24 +399,25 @@ kind: Network
 metadata:
   name: cluster
 spec:
-  clusterNetwork: <1>
-  - cidr: 10.128.0.0/14
+  clusterNetwork: 
+  - cidr: 172.24.0.0/14
```
Comment (zhaozhanqi) [inline modules/nw-operator-cr.adoc:403]: why this changes?

**Review by kelbrown20 — COMMENTED** [review]
Great work! Updates LGTM!

File: modules/nw-operator-cr.adoc [hunk 2]
```diff
@@ -52,7 +52,7 @@ The CNO configuration inherits the following fields during cluster installation
 ifdef::post-install-network-configuration,operator[]
 [NOTE]
 ====
-After cluster installation, you cannot modify the fields listed in the previous section.
+After cluster installation, you can only modify the `clusterNetwork` IP address range. The default network type can only be changed from OpenShift SDN to OVN-Kubernetes through migration.
```
Comment (danwinship) [inline modules/nw-operator-cr.adoc:55]: Link to the migration docs?

Also, looking at the preview, the text right before the list of fields says "these fields cannot be changed:" so that probably needs an update too

File: modules/nw-operator-cr.adoc [hunk 3]
```diff
@@ -372,7 +352,7 @@ defaultNetwork:
 
 [discrete]
 [id="nw-operator-cr-kubeproxyconfig_{context}"]
-=== kubeProxyConfig object configuration
+=== kubeProxyConfig object configuration (OpenShiftSDN container network interface only)
```
Comment (danwinship) [inline modules/nw-operator-cr.adoc:355]: It can be configured for some third-party network plugins as well, but I don't think we discuss third-party network plugins anywhere in the official docs (as opposed to, eg, access.redhat.com) so I guess this is correct.

File: modules/nw-operator-cr.adoc (hunk 1 above)
Comment (danwinship) [reply modules/nw-operator-cr.adoc:403]: yeah, you should leave this as it was; `10.128.0.0/14` is the default clusterNetwork value for OCP

File: modules/nw-operator-cr.adoc [hunk 4]
```diff
@@ -419,24 +399,25 @@ kind: Network
...
+  serviceNetwork:
+- 172.30.0.0/16  
+  status:
+  clusterNetwork:
+- cidr: 172.24.0.0/14
+hostPrefix: 23
+clusterNetworkMTU: 8900
+networkType: OVNKubernetes
+serviceNetwork:
+- 172.30.0.0/16
```
Comment (danwinship) [inline modules/nw-operator-cr.adoc:419]: the indentation and line order here is all mangled...

File: modules/nw-operator-cr.adoc (hunk 2 above)
Comment (ahardin-rh) [reply modules/nw-operator-cr.adoc:55]: We unfortunately can't add xref links in modules at this time, but I did add the link to the "additional resources" section

File: modules/nw-operator-cr.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/nw-operator-cr.adoc]: I reverted these changes. Thanks!
